esque/messages/avromessage.py

```diff
@@ -122,13 +130,15 @@ def decode_message_from_avro(self, message: Message):
         decoded_message = DecodedAvroMessage(
             decoded_key, decoded_value, message.partition(), key_schema_id, value_schema_id, headers=headers
         )
-        schema_version = it.count(1)
```

Comment:
We don't need to consider schema version, because schema ids are already a unique identifier for schemata in schema registry.